### DISCUSSION TITLE
Multi-domain sitemaps

### DISCUSSION THREAD

**Discussion by dvenomb98** [iamvishnusankar/next-sitemap#653] [Q&A]
how do you guys handle multi domain sitemaps?

I have for example:
domain.com/example
domain.cz/example
domain.pl/example

For dynamic content from CMS i use server-sitemap.xml that handle it well. But for static routes that comes from nextjs, i am little bit lost.

**Comment by iamvishnusankar** [comment]
Have you tried [additionalPaths](https://github.com/iamvishnusankar/next-sitemap#additional-paths-function)? I think you could add additional paths during build time based on the build environment.
